changelog.md

```diff
@@ -1,3 +1,27 @@
+1.2.0
+==========
+### New Features:
+**Microsoft Identity Web now supports App Services Authentication with Azure AD**. See https://aka.ms/ms-id-web/AppServicesAuth and issue [#8](https://github.com/AzureAD/microsoft-identity-web/issues/8) for details.
```

Comment:
Should the link use dashes (`aka.ms/ms-id-web/app-services-auth`) for consistency with previous links?